frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -500,8 +500,8 @@ action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.reset-option=Re
 action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.reset-on-remote.comment=Warning: you will lose the local commits
 action.GitMachete.BaseTraverseAction.dialog.cancel-traverse=Quit the interactive traverse
 
-action.GitMachete.BaseTraverseAction.dialog.diverged-from-parent.title=Resolve Divergence from Parent
-action.GitMachete.BaseTraverseAction.dialog.diverged-from-parent.text.HTML=\
+action.GitMachete.BaseTraverseAction.dialog.unsynchronized-to-parent.title=Resolve Out Of Sync or In Sync but Fork-point Off to Parent
```

Comment:
Hmmm I'd use the specific title depending on the case (`Resolve Out of Sync to Parent`, `Resolve In Sync to Parent but Fork Point Off`)